news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,283 @@
+    </audio>
+</div>
+
+Our second and last original track of the week was [Ruin in Cascade](https://osu.ppy.sh/beatmapsets/2318032#mania/4964418) by [kanemiko](https://osu.ppy.sh/beatmaps/artists/367). Keeping up with the tradition of featuring exciting new tracks on the tiebreaker, this year's 7K MWC was no exception! A massive collaborative effort between [Critical_Star](https://osu.ppy.sh/users/3793196), [_Stan](https://osu.ppy.sh/users/1653229), [Maxus](https://osu.ppy.sh/users/4335785), [kaulpakka](https://osu.ppy.sh/users/7115625), [ExNeko](https://osu.ppy.sh/users/7590894), and [taba2](https://osu.ppy.sh/users/7850508).
+
+But words alone cannot describe the map, so why not enjoy it in full thanks to [cheewee10](https://osu.ppy.sh/users/4477497)'s amazing play with [Noffy](https://osu.ppy.sh/users/1541323)'s eye-watering storyboard to accompany it with!
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/jDt7cJ_YiUE" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Semifinals summary
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **Indonesia** ::{ flag=ID }:: | **6** | 1 | ::{ flag=RU }:: Russian Federation |
+| BIG MAXUS | 6 | **7** | **ET BLOCKO** |
+| **Malaysia** ::{ flag=MY }:: | **6** | -1 | ::{ flag=CL }:: Chile |
+| United States ::{ flag=US }:: | 2 | **6** | ::{ flag=PE }:: **Peru** |
+| **South Korea** ::{ flag=KR }:: | **6** | 4 | ::{ flag=JP }:: Japan |
+| **Indonesia** ::{ flag=ID }:: | **6** | 0 | ::{ flag=MY }:: Malaysia |
+| **China** ::{ flag=CN }:: | **6** | 4 | ::{ flag=PH }:: Philippines |
+| **Hong Kong** ::{ flag=HK }:: | **6** | 0 | ::{ flag=DE }:: Germany |
+| Hong Kong ::{ flag=HK }:: | 2 | **6** | ::{ flag=PE }:: **Peru** |
+
+This week was nothing short of amazing! We reached a point of the tournament where the best of the best teams are being tested against one another. The matches between ::{ flag=KR }:: South Korea versus ::{ flag=JP }:: Japan and ::{ flag=CN }:: China versus ::{ flag=PH }:: the Philippines should get the point across, with all four teams being the top 4 seeded teams on this year's Qualifiers!
+
+The last two teams standing in the winners' bracket are ::{ flag=KR }:: South Korea and ::{ flag=CN }:: China, who'll be facing each other in the Finals to see who gets to keep their spot in the Grand Finals. The rest of teams however haven't had it as easily in the losers' bracket given how high the stakes were.
+
+On the other side of the bracket, teams had to win two consecutive matches to keep up with the pace. ::{ flag=ID }:: Indonesia and ::{ flag=PE }:: Peru were able to survive the gauntlet, moving on to the Finals for their very last chance at redemption. A very exciting moment for Indonesian fans, who get to see their country get to the Finals for the first time since all the way back to 2017!
```

Comment:
```suggestion
On the other side of the bracket, teams had to win two consecutive matches to keep up with the pace. ::{ flag=ID }:: Indonesia and ::{ flag=PE }:: Peru were able to survive the gauntlet, moving on to the Finals for their very last chance at redemption. A very exciting moment for Indonesian fans, who get to see their country in the Finals for the first time since all the way back in 2017!
```